src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorLanguage.ts

```diff
@@ -9,5 +9,5 @@ export class RazorLanguage {
     public static id = 'aspnetcorerazor';
     public static fileExtensions = [ 'cshtml', 'razor' ];
     public static globbingPattern = `**/*.{${RazorLanguage.fileExtensions.join(',')}}`;
-    public static documentSelector: vscode.DocumentSelector = { pattern: RazorLanguage.globbingPattern };
+    public static documentSelector: vscode.DocumentSelector = { language: this.id, pattern: RazorLanguage.globbingPattern };
```

Comment:
Just curious, does this fix anything or is it just cleanup?

Reply:
Mostly just cleanup and to avoid future bugs since we previously ran into a similar bug (https://github.com/dotnet/razor-tooling/pull/6800)